.github/workflows/livestream-docker-image.yml

```
@@ -0,0 +1,84 @@
+name: Docker CI/CD
+
+on:
+    push:
+        branches:
+            - master
+        push:
+            branches:
+                - master
+        paths:
+            - 'livestream/**'
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        permissions:
+            contents: read
+            packages: write
+
+        outputs:
+            sha: ${{ steps.push.outputs.digest }}
+
+        steps:
+            - uses: actions/checkout@v4
+            - name: Log in to the Container registry
+              uses: docker/login-action@v3
+              with:
+                  registry: ghcr.io
+                  username: ${{ github.actor }}
+                  password: ${{ secrets.GITHUB_TOKEN }}
+
+            - name: Extract metadata (tags, labels) for Docker
+              id: meta
+              uses: docker/metadata-action@v5
+              with:
+                  images: ghcr.io/posthog/livestream
+
+            - name: Set up Docker Buildx
+              uses: docker/setup-buildx-action@v2
+
+            - name: Set up QEMU
+              uses: docker/setup-qemu-action@v2
+
+            - name: Build and push Docker image
+              id: push
+              if: github.ref == 'refs/heads/master'
+              uses: docker/build-push-action@v5
+              with:
+                  context: .
+                  file: ./Dockerfile
```

at risk of not understanding things, I think this will bork 

```suggestion
                  file: ./livestream/Dockerfile
```

---

hah you suggested this right as I updated it :P

---

actually, you are right! I forgot to set the context location and should keep the file constant